.github/workflows/main.yml

```diff
@@ -0,0 +1,39 @@
+name: Build and push Docker image
+
+on: push
+
+jobs:
+  build-and-push:
+    name: Push Docker image to Docker Hub
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.4
+      - name: Docker Setup QEMU
+        uses: docker/setup-qemu-action@v1.0.1
+      - name: Docker Setup Buildx
+        uses: docker/setup-buildx-action@v1.0.3
+      - name: Docker Login
+        uses: docker/login-action@v1.6.0
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Build Docker tags
+        id: pre
+        run: |
+          if [[ $GITHUB_REF == refs/tags/* ]]; then
+            TAG="${GITHUB_REF#refs/tags/}"
+          elif [[ $GITHUB_REF == refs/heads/* ]]; then
+            TAG="${GITHUB_REF##*/}.${GITHUB_SHA}"
+          fi
+          echo ::set-output name=tag::${TAG}
+      - name: Build and push Docker images
+        uses: docker/build-push-action@v2.2.0
+        with:
+          file: core/chainlink.Dockerfile
+          tags: willianpaixao/chainlink:${{ steps.pre.outputs.tag }}
+          build-args: |
+            COMMIT_SHA=${{ github.sha }}
+            ENVIRONMENT=release
+          platforms: linux/386,linux/amd64,linux/arm64
+          push: true
```

Comment:
This should be set to false for now, we're not currently using GHA to push anything, just keeping it around for when we migrate over

Reply:
Hey @HenryNguyen5, thanks for the review. As I commented above, some values were temporary so I could push and test the build. https://github.com/smartcontractkit/chainlink/pull/3614#issuecomment-732492420.
Since the overall changes are approved, I'll update the branch.

@HenryNguyen5 will you be able to merge this PR or another one will be created?

Reply:
Yeah thanks for testing it! Please update the branch. When it's ready I will have to fork it and open it again

Reply:
@HenryNguyen5 I think I will wait until the pipeline it's fixed so we can merge the original PRs.